### PR TITLE
Added Huber three epoch model

### PR DESCRIPTION
**Pull request by noscode**
I have added three epoch model from Huber et al. 2018.

### CODE REVIEW

File: tests/test_athaliana.py [hunk 1]
```diff
@@ -46,3 +46,11 @@ class TestHuberTwoEpoch(unittest.TestCase, test_models.QcdModelTestMixin):
     """
     model = stdpopsim.arabidopsis_thaliana._HuberTwoEpoch()
     qc_model = arabidopsis_thaliana_qc.HuberTwoEpoch()
+
+
+class TestHuberThreeEpoch(unittest.TestCase, test_models.QcdModelTestMixin):
```
Comment (jeromekelleher) [inline tests/test_athaliana.py]: We should leave this out until the QC is done, otherwise CI will fail.

**Comment by jeromekelleher** [comment]
See #228 for initial issue.

**Comment by jeromekelleher** [comment]
Also, could we get a CSV parameter table like one [this](https://raw.githubusercontent.com/popgensims/stdpopsim/master/docs/parameter_tables/dromel/ooa_2.csv) for the catalog documentation please? The name needs match the species ID/model ID.

**Comment by andrewkern** [comment]
oops looks like codecov is complaining about coverage. we will probably need some tests in here. test example from the two epoch model is here: https://github.com/popgensims/stdpopsim/blob/389d506f43401dbc1c713ceb9d84c9484d6f5c02/tests/test_athaliana.py#L43

**Comment by noscode** [comment]
I have put test back without test for qc, it is the same as for two epoch model. However codecov still complains.

**Comment by jeromekelleher** [comment]
Sorry, this was my bad asking you to delete the tests @noscode. What we need to do is to inherit from ``ModelTestMixin`` rather than ``QcdModelTestMixin``, .i.e,

```python
 class TestHuberThreeEpoch(unittest.TestCase, test_models.ModelTestMixin): 
```

**Comment by noscode** [comment]
Okay, got it

**Comment by andrewkern** [comment]
looks good. one last step now-- rebase this against upstream master and squash your commits 

`git rebase -i upstream/master`

then we will merge

**Comment by jeromekelleher** [comment]
Just in case you're not familiar with rebasing @noscode, we have a guide [here](https://stdpopsim.readthedocs.io/en/latest/development.html#rebasing).

**Comment by jeromekelleher** [comment]
We'd like to get this merged soon if possible @noscode --- would you mind doing the rebase please? Alternatively, I can do the rebase, if you prefer (but I'll need to write to your branch on GitHub)

**Comment by noscode** [comment]
I am sorry but I have already done it. I checked the guide and that was exactly what I have done, I tried this one:
`git log --decorate --oneline --graph`
and it looks also fine for me. Am I not right?

**Comment by jeromekelleher** [comment]
This repo moves quite quickly @noscode, and a lot has changed in the upstream/master branch since you rebased. If you check the [network](https://github.com/popgensims/stdpopsim/network) page you'll see that your branch is a long way behind the tip of master. That's where we want it to point.

If you run ``git fetch upstream`` and then rebase, it should work fine.

**Comment by noscode** [comment]
Oh, I see, now is it okay?

**Comment by noscode** [comment]
CI have failed, but I have nothing to do with it.

File: stdpopsim/catalog/arabidopsis_thaliana.py [hunk 2]
```diff
@@ -177,3 +177,47 @@ def __init__(self):
...
+                                            metadata=self.populations[0].asdict())]
+        self.demographic_events = [
+            msprime.PopulationParametersChange(
+                time=t_3, initial_size=N_2, population_id=0),
+            msprime.PopulationParametersChange(
+                time=t_2 + t_3, initial_size=N_A, population_id=0)]
+        self.migration_matrix = [[0]]
+
+
+_species.add_model(_HuberTwoEpoch())
```
Comment (jeromekelleher) [inline stdpopsim/catalog/arabidopsis_thaliana.py]: This needs to be ``_HuberThreeEpoch``

**Comment by jeromekelleher** [comment]
Just spotted a typo which only got caught because of recent changes. Good to merge once that's fixed.

(If you do ``git commit -a --amend`` after you've fixed the typo, you can avoid needing to squash and rebase. You'll still need to force-push your branch.)

**Comment by noscode** [comment]
I believe the next time I will be faster... :)

**Comment by jeromekelleher** [comment]
Fantastic, thanks @noscode!

**Pull request merged**